CONTRIBUTING.md

```diff
@@ -597,23 +597,21 @@ look in particular at the "Writing" and "Fixing" columns).
 Contributing to the standard library is also made easier by not having
 to learn about Coq's internals, and its implementation language.
 
-However, you should be aware that, because of the age of the library,
-and the compatibility constraints created by the many projects that
+Due to the compatibility constraints created by the many projects that
 depend on it, proposing breaking changes, such as changing a
 definition, may frequently be rejected, or at the very least might
-take a long time before getting approved and merged.  This does not
-mean that you cannot try.
+take a long time before getting approved and merged. This does not
+mean that you cannot try.  On the other hand, contributing new lemmas
+on existing definitions should be generally favorably accepted,
+cleaning up existing proofs as well.  Contributing new operations on
```

Comment:
On the other hand, contributing new lemmas
on existing definitions and cleaning up existing proofs are likely to be accepted.